renderdoc/driver/vulkan/vk_debug.cpp

```diff
@@ -4701,6 +4720,76 @@ void VulkanDebugManager::MakeGraphicsPipelineInfo(VkGraphicsPipelineCreateInfo &
   pipeCreateInfo = ret;
 }
 
+void VulkanDebugManager::MakeComputePipelineInfo(VkComputePipelineCreateInfo &pipeCreateInfo,
+                                                 ResourceId pipeline)
+{
+  const VulkanCreationInfo::Pipeline &pipeInfo = m_pDriver->m_CreationInfo.m_Pipeline[pipeline];
+
+  VkPipelineShaderStageCreateInfo stage;    // Returned by value
+  static VkSpecializationInfo specInfo;
+  static vector<VkSpecializationMapEntry> specMapEntries;
+
+  const uint32_t i = 5;    // Compute stage
+  RDCASSERT(pipeInfo.shaders[i].module != ResourceId());
+
+  size_t specEntries = 0;
+  if(!pipeInfo.shaders[i].specialization.empty())
+    specEntries += pipeInfo.shaders[i].specialization.size();
+
+  specMapEntries.resize(specEntries);
+  VkSpecializationMapEntry *entry = &specMapEntries[0];
+
+  stage.sType = VK_STRUCTURE_TYPE_PIPELINE_SHADER_STAGE_CREATE_INFO;
+  stage.stage = (VkShaderStageFlagBits)(1 << i);
+  stage.module = GetResourceManager()->GetCurrentHandle<VkShaderModule>(pipeInfo.shaders[i].module);
+  stage.pName = pipeInfo.shaders[i].entryPoint.c_str();
+  stage.pNext = NULL;
+  stage.pSpecializationInfo = NULL;
+
+  if(!pipeInfo.shaders[i].specialization.empty())
```

Comment:
My app didnt have any specilization data, but the logic is copied from the original function.